packages/storybook8/stories/Composites/TypeScriptLoading/Docs.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Composites/JavaScript Loaders" />
+
+# Using the UI Composites in other Frameworks
+
+The Azure Communication Services UI library is written in [React](https://react.dev/). If you are using a different framework,
+you can still use the UI Composites by using our Loader functions for each of the composites.
+
+These functions mount each of our React Composites to your applications DOM. The Functions that are available are:
+
+- `loadChatComposite`
+- `loadCallComposite`
+- `loadCallWithChatComposite`
+- `loadOutboundCallComposite`
+
+## How they work
+
+Since all different [Node](https://nodejs.org/en) frameworks are just JavaScript, CSS, and HTML you can use each of the
+different frameworks inside each other. In the case of using [React](https://react.dev/) inside your application you would use
+the `createRoot` function to attach a React node to your application and then use your react component inside that.
+
+Our loader functions work in a similar way. They take a DOM element and attach the React Composite to that element. What this means
+is that you can use the UI Composites in any framework that can render HTML elements to the DOM.
+
+**Note:** Something to keep in mind when using these loader functions for the UI Composites is that you are taking on
+not just the bundle for `@azure/communication-react` but also the bundle for `react` and `react-dom`. Best practice here
+is to lazy load these chunks so that you don't get caught waiting for the large bundle when you don't need your
```

Comment:
This is a tad confusing as we aren't explaining how to lazy load or the fact that internally we have made the react loading async. Could this be reworded more to indicate that react and react-dom uses are set up for lazy loading and using async imports?